modules/tinymce/src/core/main/ts/caret/CaretPosition.ts

```diff
@@ -74,7 +73,7 @@ const getBrClientRect = (brNode: Element): ClientRect => {
 };
 
 // Safari will not return a rect for <p>a<br>|b</p> for some odd reason
-const getBoundingClientRectWebKitText = (rng: Range): ClientRect => {
+const getBoundingClientRectWebKitText = (rng: Range): GeomClientRect.ClientRect | null => {
```

Comment:
Should we Jira up renaming this (if it's not public) so it doesn't conflict with the DOM type?

Reply:
It is public unfortunately...